pyproject.toml

```diff
@@ -15,11 +15,10 @@ skip_magic_trailing_comma = true  # For compatibility with pydoc>=4.6, check if
 max-line-length=120
 disable = [
 
-  # Warnings
+  # To keep
   "fixme",
-
-  # Info
   "c-extension-no-member",
+  "too-many-public-methods",   # Or raise the limit from 20
```

Comment:
Where does that come from?

Reply:
`Pipeline`: it has now 23 public methods if I remember correctly :sweat_smile: I have the impression that the deepset cloud methods could be moved into a very shallow wrapper, for readability, but that's definitely for another PR.

I was thinking about a method like `Pipeline.to_dc_pipeline()` that returns a simple `DeepsetCloudPipeline` object that exposes those static methods. This pipeline would be a composition over a regular `Pipeline`, so all pipeline methods would work directly (`DeepsetCloudPipeline.add_node()` for example). If you have some free time it could be a nice improvement to reduce the size of `Pipeline` a bit, but far from urgent.